testcases/android/master.jinja2

```diff
@@ -0,0 +1,107 @@
+{% extends "master.jinja2" %}
+
+{# timeouts #}
+{% set timeout_download = timeout_download|default(0) %}
+{% set timeout_deploy = timeout_deploy|default(25) %}
+{% set timeout_between_deploy_boot = timeout_between_deploy_boot|default(0) %}
+{% set timeout_boot = timeout_boot|default(10) %}
+{% set timeout_common_tests = timeout_common_tests|default(20) %}
+{% set timeout_tests = timeout_tests|default(10) %}
+{% set timeout_job_total = timeout_download + timeout_deploy + timeout_between_deploy_boot + timeout_boot + timeout_common_tests + timeout_tests %}
+
+{# urls #}
+{% set url_docker = url_docker|default("yongqinliu/linaro-android-docker:0.1") %}
+{% set url_test_defintions_repo = url_test_defintions_repo|default("https://github.com/Linaro/test-definitions.git") %}
+
+{% set job_name = job_name|default("base") %}
+
+{# variables needs to be converted to be used by master.jinja2 #}
+{% set lava_job_name = "lkft-android-" + KERNEL_BRANCH + '-' + BUILD_NUMBER + '-' + job_name %}
+{% set lava_device_type = lava_device_type %}
+{% set LAVA_JOB_PRIORITY = TEST_LAVA_JOB_PRIORITY %}
+{% set job_timeout = timeout_job_total %}
+
+{########## LAVA JOB DEFINITION ##############}
+{% block visibility %}
+  group:
+    - {{TEST_LAVA_JOB_GROUP}}
+{% endblock visibility %}
+
+{% block metadata %}
+  android.build: "{{BUILD_NUMBER}}"
+  android.name: "{{JOB_NAME}}"
+  android.url: "{{REFERENCE_BUILD_URL}}"
+  android.version: "{{ANDROID_VERSION}}"
+{% if ANDROID_VENDOR_FINGERPRINT is defined %}
+  android.build.vendor.fingerprint: "{{ANDROID_VENDOR_FINGERPRINT}}"
+{% endif %}
+{% if ANDROID_GSI_FINGERPRINT is defined %}
+  android.build.gsi.url: "{{ANDROID_GSI_URL}}"
+  android.build.gsi.fingerprint: "{{ANDROID_GSI_FINGERPRINT}}"
+{% endif %}
+{% if TUXSUITE_DOWNLOAD_URL is defined %}
+  tuxsuite.download.url: "{{TUXSUITE_DOWNLOAD_URL}}"
+{% endif %}
+  lkft.build.config: "{{LKFT_BUILD_CONFIG}}"
+  git branch: "{{KERNEL_BRANCH}}"
+  git repo: "{{KERNEL_REPO}}"
+  git commit: "{{KERNEL_COMMIT}}"
+  git describe: "{{KERNEL_DESCRIBE}}"
+  build-url: "{{BUILD_URL}}"
+  toolchain: "{{TEST_METADATA_TOOLCHAIN}}"
+  series: lkft
+{% block job_specific_metadata %}
+{% endblock job_specific_metadata %}
+{% endblock metadata %}
+
+{% block secrets %}
+secrets:
+  ARTIFACTORIAL_TOKEN: "{{ARTIFACTORIAL_TOKEN}}"
+{% endblock %}
+
+actions:
+{% block actions %}
```

Comment:
can't 'actions' from master.jinja2 be used

Reply:
Android needs its own blocks to be defined under the actions, if not defined here, how to define?

Reply:
can you share a job definition file from LAVA so I can understand where this have to be added a second time?

Reply:
@roxell you mean the L62  or the lines after L63? L62 is not necessary, i will remove it. in the generated lava job definition, there is only one "actions:". For L63, it defined the blocks that android jobs need to overwrite, like here: https://lkft.validation.linaro.org/scheduler/job/4772849/definition

Reply:
in the new update, removed the "actions:" line here.

Reply:
you should be use '{{ super() }}' inside the '{% block actions %}'

Reply:
the actions defined here are common actions, and concept of the actions is too big.